mlir/utils/jenkins/MIOpenDriver.py

```diff
@@ -273,17 +274,25 @@ def runConfigWithMLIR(config: ConvConfiguration):
         outs, errs = p3.communicate()
 
 def runConfigWithMIOpenDriver(commandLine, envs):
-    # remove the result file generated by rocprof in previous benchmarking
-    os.system("rm "+BENCHMARKING_RESULT_FILE_NAME)
-    profilerCommand = [ROCPROF, '--stats', MIOPEN_DRIVER, *commandLine, '-V', '0']
+    MIOpenDriverCommand = [MIOPEN_DRIVER, *commandLine, '-V', '0']
     print("Running MIOpen Benchmark: ", ' '.join(commandLine))
-    # invoke rocprof + MIOpenDriver.
-    p1 = subprocess.Popen(profilerCommand, stdout=subprocess.PIPE, stderr=subprocess.PIPE, env=envs)
+    # invoke MIOpenDriver.
+    p1 = subprocess.Popen(MIOpenDriverCommand, stdout=subprocess.PIPE, stderr=subprocess.PIPE, env=envs)
     # get output.
     try:
         outs, errs = p1.communicate(timeout=180)
         if len(errs) > 0:
             print("MIOpen benchmark produced errors: ", errs.decode('utf-8'))
+            return np.nan
+        else:
+            # convert bytes to str
+            outs=outs.decode('utf-8')
+            # Extract Elapsed time in ms from the output of MIOpenDriver
+            # Use regular expression to match the contents between
+            # "Elasped: " (note the space at the end) and "ms"
+            pattern = re.compile(r"Elapsed: (.*)ms");
```

Comment:
Ok, actually, I know I just said "approved", but one minor point before this goes in if possible - move this to a global constant so it only gets compiled once.